apps/govern/hooks/useNomineesWeights.ts

```diff
@@ -2,17 +2,85 @@ import { ethers } from 'ethers';
 import { Abi } from 'viem';
 import { Address } from 'viem';
 import { mainnet } from 'viem/chains';
-import { useReadContracts } from 'wagmi';
+import { useReadContract, useReadContracts } from 'wagmi';
 
-import { VOTE_WEIGHTING } from 'libs/util-contracts/src/lib/abiAndAddresses';
+import { TOKENOMICS, VOTE_WEIGHTING } from 'libs/util-contracts/src/lib/abiAndAddresses';
+import { useMemo } from 'react';
 
 type Weight = { percentage: number; value: number };
 
+const TOKENOMICS_CONTRACT = {
+  address: TOKENOMICS.addresses[mainnet.id],
+  abi: TOKENOMICS.abi,
+  chainId: mainnet.id,
+} as const;
+
+const useStakingInflation = () => {
```

Comment:
@kupermind the formula you provided is used here, let me know if anything is unclear